### PR TITLE
Add security section for roles definition

### PR DESCRIPTION
**Pull request by ponceta**
Fix https://github.com/QGEP/docs/issues/103 

Add users role definition.

### CODE REVIEW

**Comment by ponceta** [comment]
@sjib This is a first proposal, could be enhanced if you see something missing.

**Comment by ponceta** [comment]
> What does mean "Has the highest privileges" - what does this mean. The others seem clear to me: they have different edit and view privileges

@sjib would you prefer to mention that he has access on qgep_sys shema?

**Comment by ponceta** [comment]
About qgep_sysadmin user :
* qgep_sysadmin : superuser of QGEP database, can edit qgep system tables (`qgep_sys` schema).

**Comment by sjib** [comment]
@m-kuhn can we merge this?

File: en/admin-guide/security/index.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+User Roles
+=======
+
+QGEP comes with a set of predefined roles for a multi user configuration.
+
+The SQL related to these roles creation is stored here : 
+
+https://github.com/QGEP/datamodel/blob/master/12_roles.sql
+
+Users are defined in these classes :
```
Comment (m-kuhn) [inline en/admin-guide/security/index.rst:12]:
```suggestion
The base installation of QGEP defines the following group roles :
```

File: en/admin-guide/security/index.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.. _security:
+
+User Roles
+=======
+
+QGEP comes with a set of predefined roles for a multi user configuration.
+
+The SQL related to these roles creation is stored here : 
+
+https://github.com/QGEP/datamodel/blob/master/12_roles.sql
```
Comment (m-kuhn) [inline en/admin-guide/security/index.rst:10]: I think this is "advanced reading". Most will be happy with the descriptive list. I would therefore put this at the end of the section, maybe even in a `.. note::` block

**Comment by ponceta** [comment]
@sjib merge?

**Pull request merged**